build.gradle

```diff
@@ -56,12 +56,6 @@ allprojects {
       downloadSources = false
     }
   }
-
-  tasks.withType(JavaForkOptions).configureEach {
-    maxHeapSize = System.properties["otel.forkedMaxHeapSize"]
-    minHeapSize = System.properties["otel.forkedMinHeapSize"]
-    jvmArgs "-XX:ErrorFile=/tmp/hs_err_pid%p.log"
```

Comment:
Nice find!

Reply:
I only wondering if we may still need that heap config...

Reply:
a, well, let us see what will happen. Experiment FTW!